recipes/pyunlocbox/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pyunlocbox" %}
+{% set version = "0.5.1" %}
+{% set sha256 = "50e6e900fe321e6335a6cf71854156075650068bd75980a33f1ae3b83215f5ca" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
```

Comment:
It looks like `numpy` and `scipy` are not `build` requirements since they are not imported by the `setup.py` directly or by side-effect. The `--single-version-externally-managed` allows you to ignore what is in `install_requires` for the `build` reqs.

Reply:
Thanks, will remove `numpy` and `scipy`.